content/en/docs/started/installing-kubeflow.md

```diff
@@ -5,243 +5,255 @@ weight = 20
 
 +++
 
+## What is Kubeflow?
+
 Kubeflow is an end-to-end Machine Learning (ML) platform for Kubernetes, it provides components for each stage in the ML lifecycle, from exploration through to training and deployment.
 Operators can choose what is best for their users, there is no requirement to deploy every component.
-To read more about the components and architecture of Kubeflow, please see the <a href="/docs/started/architecture/">Kubeflow Architecture</a> page.
+Learn more about Kubeflow in the [Introduction](/docs/started/introduction/) and
+[Architecture](/docs/started/architecture/) pages.
 
 ### How to install Kubeflow?
 
-There are two pathways to get up and running with Kubeflow, you may either:
+Anywhere you are running Kubernetes, you should be able to run Kubeflow.
+There are two primary ways to install Kubeflow:
 
-1. [Use a packaged distribution](#packaged-distributions-of-kubeflow)
-2. [Use the raw manifests](#raw-kubeflow-manifests) <sup>(for advanced users)</sup>
+1. [**Packaged Distributions**](#packaged-distributions-of-kubeflow)
+1. [**Raw Manifests**](#raw-kubeflow-manifests) <sup>(advanced users)</sup>
 
 <a id="packaged-distributions"></a>
 <a id="install-a-packaged-kubeflow-distribution"></a>
-## Packaged Distributions of Kubeflow 
+
+## Packaged Distributions of Kubeflow
+
+Packaged distributions are maintained by various organizations and typically aim to provide
+a simplified installation and management experience for Kubeflow.
 
 {{% alert title="Note" color="warning" %}}
-Packaged distributions are developed and supported by their respective maintainers, <b>the Kubeflow community does not endorse or certify any specific distribution</b>.
+Packaged distributions are developed and supported by their respective maintainers.
+The Kubeflow community <strong>does not endorse or certify</strong> any specific distribution.
 
 In the near future, there are plans to introduce <a href="https://github.com/kubeflow/community/blob/master/proposals/kubeflow-conformance-program-proposal.md">conformance testing for distributions</a>, you may track progress on this initiative by following <a href="https://github.com/kubeflow/kubeflow/issues/6485">kubeflow/kubeflow#6485</a>.
 {{% /alert %}}
 
-### Active Distributions
+Some distributions can be deployed on all certified Kubernetes
+distributions <sup>[<a href="https://kubernetes.io/partners/#conformance">1</a>]</sup>, some of them
+can be deployed only in specific Kubernetes environment (e.g. EKS or GKE).
```

Comment:
If we are going to make the distinction between distributions for all Kubernetes, and ones for only a specific platform, I think it makes sense to split the tables up into "generic" and "specific" with different columns:

__"Generic" Table:__

```md
Some Kubeflow distributions can be deployed on [all certified Kubernetes
distributions](https://kubernetes.io/partners/#conformance):
```

Distribution Name | Kubeflow Version | Maintainer | Links
--- | --- | --- | ---
Charmed Kubeflow | 1.8.0 | Canonical | Website
deployKF | 1.7.0 | Aranui Solutions | Website

---

__"Specific" Table:__

```md
Other Kubeflow distributions target a specific Kubernetes platform:
```

Platform | Kubeflow Version | Maintainer | Links
--- | --- | --- | ---
Google Kubernetes Engine (GKE) | 1.8.0 | Google Cloud | Website
IBM Cloud Kubernetes Service (IKS) | 1.8.0 | IBM Cloud | Website
... | ... | ... | ...

Reply:
Yes, we discussed this before, so your suggestion @thesuperzapper is to name Distributions as:
- Genetic Distributions
- Specific Distributions

What do others think about it @ca-scribner @kubeflow/kubeflow-steering-committee @juliusvonkohout ?

Reply:
Probably yes. Which table should go first? Maybe the specific ones?

Reply:
I will raise this question on tomorrow's call.

Reply:
@andreyvelich I think putting the "generic" table first is more in the spirit of Kubeflow as a "generic" platform.

I worry that listing the "specific" distributions first will imply that the first platform on that table is the "recommended platform for Kubeflow". (We are already trying to shake the perception that Kubeflow is a Google-focused product).

Also, users will already gravitate toward the distribution targeting their specific platform, so it just doubly disadvantages the generic distributions by putting them second.

However, I understand that I may be a little biased (I maintain deployKF), so I am happy to see what others think.

Reply:
Based on today's community meeting, I think the consensus was to keep a single table, and to re-add the "Distribution Name" column (but keep it empty for distributions initially, until they make a PR to update the website with a non `"Kubeflow on XXXX"` name.

See my comment here for more details: https://github.com/kubeflow/website/pull/3693#discussion_r1513480637